README.md

```
@@ -142,19 +142,19 @@ The generated Wasm module is available in the file `out/out_Linux_x86_64.wasm`.
 Create and install the canister with:
 
 ```
-dfx canister --ic install <canister_id> --wasm out/out_Linux_x86_64.wasm --argument="(opt principal \"cngnf-vqaaa-aaaar-qag4q-cai\", opt principal \"2vxsx-fae\")"
+dfx canister --ic install <canister_id> --wasm out/out_Linux_x86_64.wasm --argument="(opt principal \"cngnf-vqaaa-aaaar-qag4q-cai\", opt principal \"2vxsx-fae\", opt principal \"6jrls-gqaaa-aaaao-a4pgq-cai\")"
 ```
 
 #### Reinstall
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)